scripts/install-lib

```diff
@@ -33,7 +33,8 @@ is-enabled() {
 #
 # List of conflicting units
 get-conflicts() {
-    systemctl show "$1" | awk -F'=' '/^Conflicts=/{print $2}'
+    systemctl show "$1" | awk -F'=' '/^Conflicts=/{print $2}' \
+        | awk 'BEGIN { ORS=" " }; { for(i = 1; i <= NF; i++) if($i != "shutdown.target") print $i }'
```

Comment:
```suggestion
        |  sed 's|\bshutdown.target\b||'
```

Reply:
Didn't think that sed could do it in the sense of not interrupting a space separated list. If that works as well, I'm all for the change. Please review this as I'm blindly committing that from my phone (pc is shut down for today).

Reply:
Just testee it and, as guessed, it keeps both spaces where the shutdown.target was supposed to be.

Not the cleanest, but our purpose of looping over it doesn't care about it.

Basically the change would turn `a b shutdown.target c` into `a b  c` instead of `a b c` with awk.